schemas/webhooks/definitions.yaml

```diff
@@ -84,7 +84,6 @@ webhook_event_type:
     - report.awaiting_approval
     - report.completed
     - workflow_timeline_file.created
-    - workflow_signed_evidence_file.created
```

Comment:
Aren't we adding the missing resource types too?

Reply:
Probably you're referring at the change below?
https://github.com/onfido/onfido-openapi-spec/pull/138/files

It has been split in two as the other change is not breaking backward compatibility.